packages/webapp/src/members/api/members.ts

```diff
@@ -28,13 +28,21 @@ export const getMember = async (
 };
 
 export const getMembers = async (
-    page = 1,
-    limit = 200,
+    page: number,
+    limit: number,
```

Comment:
Since it's a bit non-obvious. These defaults were pushed out a level so we could properly label the cached react-query values in the case where these defaults were used. See queries.ts L86 for where this moved to https://github.com/eoscommunity/Eden/pull/377/files#diff-a9fbc3a9c85acfa16e0e831f068c0e7aeada0af1932a6c756eae4e795866155fR86